src/main/scala/dgalkin/hw_02/Task01.sc

```diff
@@ -0,0 +1,110 @@
+// task 01
+/*
+The signum of a number is 1 if the number is positive, –1 if it is negative, and 0 if it is zero.Write a function that computes this value.
+*/
+
+def signum(x: Int) = if (x > 0) 1 else if (x == 0) 0 else -1
+signum(1)
+signum(-1)
+signum(0)
+
+
+// task 02
+/*
+What is the value of an empty block expression {}? What is its type?
+*/
+val test = {}
+// val test: Unit = ()
+
+// task 03
+//Come up with one situation where the assignment
+//x = y = 1 is valid in Scala. (Hint: Pick asuitable type for x.)
+
+var y = 1
+var x = y = 1
```

Comment:
Let's assume that you have already created var x. What should be the type of x to make 
var x = y = 1 legal?